php/BUILD.bazel

```diff
@@ -137,3 +139,48 @@ pkg_files(
     strip_prefix = strip_prefix.from_pkg(""),
     visibility = ["//pkg:__pkg__"],
 )
+
+################################################################################
+# PECL .tgz Release
+################################################################################
+
+pkg_files(
+  name = "utf8_range_files",
+  srcs = [
+    "@utf8_range//:utf8_range_srcs",
+    "@utf8_range//:LICENSE",
+  ],
+  prefix = "third_party/utf8_range",
+)
+
+pkg_files(
+  name = "protobuf_files",
+  srcs = glob([
+    "ext/google/protobuf/*.h",
+    "ext/google/protobuf/*.c",
+  ]) + [
+    "//:LICENSE",
+    "ext/google/protobuf/config.m4",
+    "ext/google/protobuf/wkt.inc",
+  ],
+)
+
+pkg_filegroup(
+  name = "release_files",
+  srcs = [
+    ":protobuf_files",
+    ":utf8_range_files",
+  ],
+  prefix = "protobuf-%s" % PROTOBUF_PHP_VERSION,
+)
+
+pkg_tar(
+  name = "release",
+  extension = "tgz",
+  srcs = [
+    ":release_files",
+    "ext/google/protobuf/package.xml",
+  ],
+  package_file_name = "protobuf-%s.tgz" % PROTOBUF_PHP_VERSION,
```

Comment:
Note that the PHP release version that we write into package.xml (and thus the .tgz) doesn't quite match the format we use for protobuf_version.bzl for other versions. For PHP we use 3.21.0RC1 instead of 3.21.0-rc1. Maybe its ok to keep this in protobuf_version.bzl, but we should ensure it reflects the correct version.

I have logic in the package.xml generation to read the version number directly from version.json right now and reformat, but I think having a PROTOBUF_PHP_VERSION is probably better.

Reply:
I think we should be able to keep PROTOBUF_PHP_VERSION to be the correct formatting, even if that means not all the formats in protobuf_version.bzl match.